src/components/ContactCard/ContactFormField.jsx

```diff
@@ -32,17 +44,20 @@ class ContactFieldInput extends React.Component {
       withLabel,
       labelPlaceholder
     } = this.props;
-    const { renderLabel } = this.state;
+    const { renderLabel, hasFocus } = this.state;
+    const focusedClass = hasFocus
+      ? " contact-form__input-wrapper--focused"
+      : "";
 
     return (
-      <div className="contact-form__input-wrapper">
+      <div className={`contact-form__input-wrapper${focusedClass}`}>
```

Comment:
[Not blocking] Not fond of this. I recommend https://github.com/JedWatson/classnames , even if the logic is very basic